index.js

```diff
@@ -40,7 +39,51 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header")
+
+function handleToggleColor(e) {
+  toggleColor(e.target)
+}
+
+header.addEventListener('click', handleToggleColor)
+
+
 
 /***** Deliverable 2 *****/
+const newPlayerForm = document.querySelector("#new-player-form")
+
+function handlePlayerFormSubmit(e) {
+  e.preventDefault()
+  
+  const number = e.target.number.value
+  const name = e.target.name.value
+  const nickname = e.target.nickname.value
+  const photo = e.target.photo.value
+  
+  const playerObj = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 999
+  }
```

Comment:
Looks good! If you want to tighten the code up a bit, you could create your object with the values from the input fields directly (just a stylistic choice): 

```js
  const playerObj = {
    number: e.target.number.value,
    name: e.target.name.value,
    nickname: e.target.nickname.value,
    photo: e.target.photo.value,
    likes: 999
  }
```